libs/utils/utils.js

```diff
@@ -304,6 +304,29 @@ export async function loadTemplate() {
   await Promise.all([styleLoaded, scriptLoaded]);
 }
 
+export async function universalButtons(links) {
```

Comment:
as this function decorates links as buttons, I'd use a more precise name and use a verb, as e.g. `decorateLinksAsButtons`

Reply:
good point, thanks @jckautzmann!